tutorials/dapt-curation/code/main.py

```diff
@@ -0,0 +1,245 @@
+# +
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import shutil
+from typing import Any, Optional, Tuple
+
+import matplotlib.pyplot as plt
+from downloaders import (
+    download_github_sources,
+    download_pdf_sources,
+    download_wikipedia_sources,
+)
+from utils import (
+    clean_and_unify,
+    dedupe,
+    filter_code,
+    filter_code_dataset,
+    filter_code_lines,
+    filter_txt,
+    filter_txt_lines,
+    redact,
+    redact_pii,
+)
+
+import nemo_curator as nc
+from nemo_curator import ScoreFilter, Sequential
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.filters import RepeatingTopNGramsFilter, WordCountFilter
+from nemo_curator.modifiers.pii_modifier import PiiModifier
+from nemo_curator.modifiers.unicode_reformatter import UnicodeReformatter
+from nemo_curator.modules import ExactDuplicates
+from nemo_curator.modules.modify import Modify
+from nemo_curator.utils.distributed_utils import get_client
+from nemo_curator.utils.file_utils import (
+    get_all_files_paths_under,
+    separate_by_metadata,
+)
+from nemo_curator.utils.script_utils import add_distributed_args, parse_client_args
+
+SCRIPT_DIR_PATH = os.path.dirname(os.path.abspath(__file__))
+DATA_DIR = os.path.join(SCRIPT_DIR_PATH, "data")
+
+#
+# TODO
```

Comment:
Could you add a note about this in the README file?

Reply:
done

Reply:
Did you also remove the TODO notes here? If already removed, it's not showing up here for some reason.